tendrl/commons/utils/ssh_util.py

```diff
@@ -0,0 +1,64 @@
+import psutil
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons.utils import cmd_utils
+
+
+def sshd_status():
+    cmd = cmd_utils.Command("systemctl show sshd.service")
+    out, err, rc = cmd.run(tendrl_ns.config.data[
+                           'tendrl_ansible_exec_file'])
+    if not err:
+        sshd = {}
+        out = out.split("\n")
+        pid = find_pid(out)
+        if pid == 0:
+            Event(
+                Message(
+                    priority="error",
+                    publisher="commons",
+                    payload={"message": "sshd service is not run"}
+                )
+            )
+            return sshd, "sshd service is not run"
```

Comment:
No returns in between. Write pre-determined status values in a constant and use one of them in the dict you return at the end of the method. You can short circuit in between if the checks further down don't make any sense, but return must always be a dict with the same format each time.